code/datums/gamemodes/nuclear.dm

```diff
@@ -48,13 +48,21 @@
 
 		if (player.ready)
 			num_players++
+#ifndef ME_AND_MY_40_ALT_ACCOUNTS
+	if (num_players < 15)
```

Comment:
this should be an in-file define like `AMOUNT_OF_VALID_NUKE_PLANT_LOCATIONS`; make sure to reference the value in the output message.

Reply:
Making a var, I don't see a good reason for a define